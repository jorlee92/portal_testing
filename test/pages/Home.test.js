// import { shallow } from 'vue-test-utils'
// import Home from '@/pages/Home.vue'

describe('Home', () => {
  // let wrapper
  // beforeEach(() => {
  //   wrapper = shallow(Home)
  // })
  // it('should render the view correctly', () => {
  //   expect(wrapper.element).toMatchSnapshot()
  // })
  xit('tests to go here', () => {})
})
