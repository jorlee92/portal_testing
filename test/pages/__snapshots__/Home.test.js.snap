// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render the view correctly 1`] = `
<div>
  <h1>
    This be a home page
  </h1>
</div>
`;
